autoload/youcompleteme.vim

```diff
@@ -23,6 +23,7 @@ set cpo&vim
 let s:script_folder_path = escape( expand( '<sfile>:p:h' ), '\' )
 let s:omnifunc_mode = 0
 
+let s:has_textchangedi = 0
```

Comment:
one last comment (I promise) why don't initialise this directly instead of setting it to `0` and then reset the value later?